courses/free-courses-en.md

```diff
@@ -1265,6 +1265,7 @@
 * [Linux Tutorials and Projects](https://www.udemy.com/course/linux-tutorials/) - Jason Cannon (Udemy)
 * [LPIC 1 exam guide: Linux Professional Institute Certification Study Guide](https://linux1st.com) - Jadi Mirmirani (HTML)
 * [LPIC 2 exam guide: Linux Professional Institute Certification Study Guide](https://borosan.gitbook.io/lpic2-exam-guide) - Payam Borosan (HTML)
+* [Practice Linux Commands (Hands-On Labs)](https://labex.io/tutorials/practice-linux-commands-free-tutorials-398420) - LabEx
```

Comment:
I think this belongs in our interactive tutorials list. Could you take a look at that list and see if you agree?

Reply:
> I think this belongs in our interactive tutorials list. Could you take a look at that list and see if you agree?

Thank you for your suggestion! I wasn't aware of the interactive tutorials list, and it is indeed a better fit. I'll relocate it to more/free-programming-interactive-tutorials-en.md. 😄